dlc-oracle/src/main/scala/org/bitcoins/dlc/oracle/DLCOracle.scala

```diff
@@ -119,6 +119,30 @@ case class DLCOracle(private val extPrivateKey: ExtPrivateKeyHardened)(implicit
     }
   }
 
+  def createNewRangedEvent(
```

Comment:
How do you handle a step where the last step is not uniform? I.e. 

start=0
step=5
stop=11

This should be well documented.